scripts/reboot

```diff
@@ -11,6 +11,8 @@ PLATFORM_UPDATE_REBOOT_CAUSE="platform_update_reboot_cause"
 REBOOT_CAUSE_FILE="/host/reboot-cause/reboot-cause.txt"
 PLATFORM_REBOOT_PRE_CHECK="platform_reboot_pre_check"
 REBOOT_TIME=$(date)
+PLATFORM_JSON_FILE="platform.json"
+PLATFORM_JSON_PATH="${DEVPATH}/${PLATFORM}/${PLATFORM_JSON_FILE}"
```

Comment:
The `${PLATFORM}` definition comes after the variable is being used [in this line ](https://github.com/sonic-net/sonic-utilities/blob/7792b3e9eb341e91b3255d189c98bfa71eb43587/scripts/reboot#L33). This has to be moved